src/js/common/utopia-card-rules.js

```diff
@@ -4999,6 +4999,18 @@ intercept: {
 	//Robinson :71213
 
 	//Dreadnought(old) :71212
+		"ship:S221": {
+			intercept: {
+				ship: {
+					canEquipCaptain: function (captain, ship, fleet) {
```

Comment:
why not just something like this here? 
intercept:
...
ship: {
 canEquipCaptain: false,
 canEquipAdmiral: false
 }
...

Wouldn't that work as well ?

Reply:
Haha, that was my original thought too. I tried it locally and it didn't prevent the user from moving a captain/admiral to the slot. I think there may be a callback function requirement somewhere?